README.md

```diff
@@ -31,11 +31,14 @@ This repo is a fork of the excellent work performed by Godson Fortil.  The origi
 1. Decide how you want to supply option values to the module during invocation.  There are three possibilities:
 	1. Invoke the `terraform apply` command and enter values for each option as Terraform prompts for it, then enter `yes` at the final prompt to begin building the cluster.
 	1. **Recommended:**  Create a `terraform.tfvars` file containing the values for each option, invoke `terraform apply`, then enter `yes` at the final prompt to begin building the cluster.  The easiest way to do that is to copy the sample file and then edit the copy:
-		* `cp examples/sample.tfvars terraform.tfvars`
+		* `cp examples/sample.tfvars terraform.tfvars` OR
+		* `cp examples/sample-with-dns-zone.tfvars terraform.tfvars`
```

Comment:
This file should be deleted after merging your changes into the original examples/sample.tfvars, so this line should be deleted as well.